projects/iio_aducm3029/src.mk

```diff
@@ -10,6 +10,5 @@ SRC_DIRS += $(PROJECT)/src \
 
 ifeq '$(TINYIIOD)' 'y'
 SRC_DIRS += $(NO-OS)/iio/iio_app
-SRC_DIRS += $(NO-OS)/drivers/irq
-SRC_DIRS += $(NO-OS)/drivers/gpio
+SRC_DIRS += $(NO-OS)/api
```

Comment:
Why not use SRC_DIRS += $(NO-OS)/api everywhere?

Reply:
As I discussed with @amiclaus , we should better include only the used files and not the whole folder. In this way, the user will know exactly which files are used by the project.